askbot/models/question.py

```diff
@@ -1571,7 +1571,9 @@ def get_last_update_info(self):
 
     def get_summary_html(self, search_state=None, visitor = None):
         html = self.get_cached_summary_html(visitor)
-        if not html:
+        if html:
+            html = html.decode('utf-8')
```

Comment:
The issue here is that caching backend in production is memcached which only stores bytes not unicode. Django is encoding the unicode on set, but doesn't decode on get (it can't know what was unicode and what was bytes).

This forces the bytes back to unicode by decoding them.